pycnite/types.py

```diff
@@ -74,8 +74,48 @@ class Opcode:
     argval: Any
 
     def __str__(self):
+        ret = f"{self.line:>5}{self.offset:>6}  {self.name:<30}"
         if self.arg is not None:
-            return (f"{self.line:>5}{self.offset:>6}  {self.name:<30}"
-                    f"{self.arg:>5} {self.argval}")
-        else:
-            return f"{self.line:>5}{self.offset:>6}  {self.name:<30}"
+            ret += f" {self.arg:>5} {self.argval}"
+        return ret
+
+
+@dataclass
+class DisassembledCode:
+    """Tree of bytecode and associated opcode list."""
+
+    code: CodeTypeBase
+    opcodes: List[Opcode]
+    children: "List[DisassembledCode]"
```

Comment:
nit: I'm pretty sure only the DisassembledCode portion needs to be quoted (here and in line 99).

Reply:
haven't we hit some corner cases in pytype with partially quoted type defs? i know i started just quoting the entire type on principle at some point

Reply:
I feel like I haven't hit any in a while. Then again, there's no harm in quoting the entire type.